frontend/src/features/projects/erp/components/ApprovalActions.tsx

```diff
@@ -49,7 +51,7 @@ export const ApprovalActions = ({
             showSubmitting
             isSubmitting={!noFetchingProjectRequests}
             onClick={() => {
-              submitForm();
+              (submitDirectly ?? submitForm)();
```

Comment:
Allow the save action to bypass formik's validation. useful for saving in-progress work without needing to have all required fields filled out.